### DISCUSSION TITLE
remark-embed-images src relative to md file?

### DISCUSSION THREAD

**Discussion by Torbet** [remarkjs/remark#966] [Q&A] [answered]
When using remark-embed-images for inline images, I have to reference the image from the root of my project, for example:

in /notes/Articles/post.md, for an image in the same folder to render, I need to reference it as !(image)[notes/Articles/Image.png]

as they are in the same folder, I would like to be able to add images relatively like: !()[./Image.png]

any ideas?

**Comment by wooorm** [comment, marked as answer]
For reference, Torbet also asked this question here: https://github.com/remarkjs/remark-embed-images/issues/7.
